### DISCUSSION TITLE
I've updated but no changes.

### DISCUSSION THREAD

**Discussion by flowerhorse** [brunoherbelin/vimix#79] [General]
Could you send me a link to your uptdating Information?
Could you also put a beta version number to 'about vimix'? (Than i could see update succes at once)

Best Regards

**Comment by brunoherbelin** [comment]
With the 0.8.0 release done, all new developments are in the Beta branch (until next release, 0.8.1, to come when ready).

To test the latest features, you can compile vimix from source with the git clone of the Beta branch, or [build local pre-release flatpack package](https://github.com/brunoherbelin/vimix/tree/master/flatpak#build-local-pre-release-flatpack-package-of-vimix).

**Comment by flowerhorse** [comment]
I've compiled a lot during last days but I didn't got any beta.

It may be that I made mistakes, because sometimes there were error messages at the beginning because of a non-existent Git directory. Then I changed the directory without knowledge until the error no longer occurred. I would like to take all Vimix versions from the computer and set up Vimix again.
Is there a simple Linux command that automates this? Or alternatively: which directories do I have to delete so that I can rebuild Vimix without overlaps?

**Comment by brunoherbelin** [comment]
It can indeed be confusing with git!

For compilation of vimix from git code, there shouldn't be anything installed outside of the build directory (i.e. the one locally in your account). (well, except if you ran 'make install' ) So to cleanup, you can simply delete all the directories with vimix. 

I will update the instructions in the [README](https://github.com/brunoherbelin/vimix/blob/master/README.md) to be easier to follow.

If your git clone is in the Beta branch ( `$ git checkout beta ` ) , if you pulled the latest git commit ( `~$ git pull `), and if you rebuild vimix ( `cmake --build .` ), you should now have the info on the git commit ID in the About vimix dialogue :

![Screenshot from 2023-04-13 23-19-29](https://user-images.githubusercontent.com/60393189/231886503-df55fe46-86f6-4329-a6e3-b417f4be65f4.png)

**Comment by flowerhorse** [comment]
Great. It worked.
